k4e-agent-sti.sh

```diff
@@ -74,32 +128,65 @@ createrepo /var/www/html/k4e-repo/
 #-----------------------------------
 git clone https://github.com/ydayagi/r4e.git /home/builder/r4e
 cd /home/builder/r4e
-./r4e-image.sh $(hostname) edgedevice helios05.lab.eng.tlv2.redhat.com:8888 http://$(hostname)/k4e-repo/
+./r4e-image.sh -s $(hostname) -i $IMAGE_NAME -o $HTTP_API -p http://$(hostname)/k4e-repo/
 
 #----------------------------------------
 # Create ISO for image and serve via http
 #----------------------------------------
-cat << EOF > /tmp/empty.toml
-name = "empty"
-description = "Empty blueprint"
+ISO_BLUEPRINT=$(mktemp)
+cat << EOF > $ISO_BLUEPRINT
+name = "edgedevice-iso"
+description = "Empty blueprint for ISO creation"
 version = "0.0.1"
 EOF
 
-if [ "$ARCH" = "aarc64" ]; then
+if [ "$ARCH" = "aarch64" ]; then
     REF="rhel/8/aarch64/edge"
 else
     REF="rhel/8/x86_64/edge"
 fi
 
-cd /var/www/html/edgedevice
-composer-cli blueprints push /tmp/empty.toml
-composer-cli compose start-ostree --ref $REF --url http://$(hostname)/edgedevice/repo/ empty edge-installer > temp.out
-BUILD_ID=$(awk '{print $2}' temp.out)
+#-------------------------
+# Save image in ISO format
+#-------------------------
+dnf install -y jq isomd5sum genisoimage
+composer-cli blueprints push $ISO_BLUEPRINT
+BUILD_ID=$(composer-cli -j compose start-ostree --ref $REF --url http://$(hostname)/$IMAGE_NAME/repo/ edgedevice-iso edge-installer | jq '.build_id')
```

Comment:
I think you need also to remove the quotas (' " ') from the that string , in order to rung the while command by using 
`| tr -d '"'`

Reply:
the quotes are helpful in the comparison of the BUILD_ID in lines 156 and 157.
i remove them in line 163 for creating the iso file name.

Reply:
I didn't see that you remove it

Reply:
removed by `ISO_FILE=${BUILD_ID//\"/}-installer.iso`